pkg/bff/api/v1/api.go

```diff
@@ -20,6 +20,7 @@ type BFFClient interface {
 	Lookup(context.Context, *LookupParams) (*LookupReply, error)
 	LookupAutocomplete(context.Context) (map[string]string, error)
 	VerifyContact(context.Context, *VerifyContactParams) (*VerifyContactReply, error)
+	GetNetworkActivity(context.Context) (*NetworkActivityReply, error)
```

Comment:
We can probably just call this `NetworkActivity`, since we won't have DELETE, PUT, etc. methods for the path. What do you think?

Reply:
that makes sense to me - I have made the change.